docs/configuration/command_line_interface.rst

```diff
@@ -161,6 +161,9 @@ These are the commands you may need to use in relation to your Mautic instance.
    * - ``mautic:segments:update``
      - Update Contacts in smart Segments based on new Contact data.
      - ``mautic:segments:rebuild``
+   * - ``mautic:segments:stat``
+     - Outputs table of all segments and whether they are being used in campaigns, emails, other segments, form actions and SMS. Useful to detect segments that can be deleted to save COGS on rebuilds.
```

Comment:
```suggestion
     - Outputs table of all segments and whether they are being used in Campaigns, Emails, other Segments, Form actions and SMS. Useful to detect Segments that can be deleted to save resources on rebuilds.
```

Folks might not be familiar with the term COGS?

Reply:
Thanks!